apps/intake/playwright.config.ts

```diff
@@ -71,7 +72,6 @@ export default defineConfig({
         ...devices['Desktop Chrome'],
         storageState: './playwright/user.json',
       },
-      dependencies: ['login'],
```

Comment:
this was added to ensure the setup step works correctly. after your changes, if you delete user.json locally without it to make it need to log in, does playwright start correctly?